diff --git a/src/systemctl/systemctl.c b/src/systemctl/systemctl.c
index 21cb898..7469801 100644
--- a/src/systemctl/systemctl.c
+++ b/src/systemctl/systemctl.c
@@ -6024,6 +6024,7 @@ static void systemctl_help(void) {
         pager_open_if_enabled();
 
         printf("%s [OPTIONS...] {COMMAND} ...\n\n"
+               " HUBBABUBBAHUBBABUBBAHUBBABUBBAHUBBABUBBAHUBBABUBBAHUBBABUBBAHUBBABUBBAHUBBABUBBAHUBBABUBBAHUBBABUBBAHUBBABUBBAHUBBABUBBAHUBBABUBBA \n\n"
                "Query or send control commands to the systemd manager.\n\n"
                "  -h --help           Show this help\n"
                "     --version        Show package version\n"
