From 0e86bd22bdc90c634dee2e5710e1c48883c76fb4 Mon Apr 4 22:01:52 2016
From: Timofey Kirillov <distorhead@gmail.com>
Date: Mon Apr 4 22:01:52 2016 +0300
Subject: hubbabubba
---
diff --git a/src/systemctl/systemctl.c b/src/systemctl/systemctl.c
index 21cb898..261beee 100644
--- a/src/systemctl/systemctl.c
+++ b/src/systemctl/systemctl.c
@@ -6024,6 +6024,28 @@ static void systemctl_help(void) {
         pager_open_if_enabled();
 
         printf("%s [OPTIONS...] {COMMAND} ...\n\n"
+               "\
+                   ooo OOO OOO ooo\n\
+               oOO                 OOo\n\
+           oOO                         OOo\n\
+        oOO                               OOo\n\
+      oOO                                   OOo\n\
+    oOO                                       OOo\n\
+   oOO                                         OOo\n\
+  oOO                                           OOo\n\
+ oOO                                             OOo\n\
+ oOO                                             OOo\n\
+ oOO                                             OOo\n\
+ oOO                                             OOo\n\
+ oOO                                             OOo\n\
+  oOO                                           OOo\n\
+   oOO                                         OOo\n\
+    oOO                                       OOo\n\
+      oOO                                   OOo\n\
+        oO                                OOo\n\
+           oOO                         OOo\n\
+               oOO                 OOo\n\
+                   ooo OOO OOO ooo\n\n"
                "Query or send control commands to the systemd manager.\n\n"
                "  -h --help           Show this help\n"
                "     --version        Show package version\n"
